src/fetch-location-lambda/fetch-location-handler.test.ts

```diff
@@ -0,0 +1,71 @@
+import axios from 'axios';
+import { handler, emptyLocationInputErrorMessage, invalidResponseObjectErrorMessage } from './fetch-location-handler';
+
+describe('fetch-location-lambda > handler', () => {
+	test('If a simple valid input is passed to the handler, it should return an object with a property "locations", filled with the locations that best fit the given input', async () => {
+		// ----- Arrange -----
+		const mockResponse = {
+			data: {
+				features: [
+					{ place_name: 'Brasília, Federal District, Brazil', center: [-47.8823, -15.7934] },
```

Comment:
Could you please help me understand your decision to go with tabs versus spaces?

Reply:
Actually, nowadays it's more like a matter of habit, since I have been using tabs for a little more than 4 years in this company that I work at. People there use to say that tabs are more "democratic" since each developer can set the tab size in his/her IDE as they wish, and I kind of agree with that 😅